packages/katello/rubygem-hammer_cli_foreman_rh_cloud/rubygem-hammer_cli_foreman_rh_cloud.spec

```diff
@@ -1,19 +1,14 @@
 # template: hammer_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
 %global gem_name hammer_cli_foreman_rh_cloud
 %global plugin_name foreman_rh_cloud
 
-%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
-%global hammer_confdir %{_root_sysconfdir}/hammer
+%global hammer_confdir %{_sysconfdir}/hammer
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.0.3
+Name: rubygem-%{gem_name}
+Version: 1.0.4
 Release: 1%{?foremandist}%{?dist}
 Summary: Foreman Rh Cloud plugin for Hammer CLI
-Group: Development/Languages
-License: GPLv3
+License: GPLv3+
```

Comment:
This comes from https://github.com/theforeman/hammer-cli-foreman-rh-cloud/commit/071ca73c065105f5734b049fa56cd2b6b61ef448, but I wonder if it's correct?

Reply:
We updated the license because of this test failure https://github.com/theforeman/hammer-cli-foreman-rh-cloud/actions/runs/14181080477/job/39727118228 so Partha changed it here to make it happy https://github.com/theforeman/hammer-cli-foreman-rh-cloud/pull/10/files#diff-c80b304344995dba0dfebab83429d8b58e7bbd54bc7cf8fb72a6c284e9a63bceR18 we have that same license in hammer-cli-katello